sycl/plugins/cuda/pi_cuda.cpp

```diff
@@ -454,13 +454,21 @@ __SYCL_EXPORT pi_result piextMemImageGetInfo(pi_image_mem_handle MemHandle,
                                      ParamValueSizeRet);
 }
 
-__SYCL_EXPORT pi_result
-piextMemImportOpaqueFD(pi_context Context, pi_device Device, size_t Size,
-                       int FileDescriptor, pi_interop_mem_handle *RetHandle) {
+__SYCL_EXPORT_DEPRECATED("This function has been deprecated in favor of "
+                         "`piextImportExternalMemory`")
```

Comment:
When do we expect to drop that? Can/should we just wait until the ABI breaking window?

Also, I'm not even sure we need this deprecated entry point. Is it only needed for old libsycl.so that will use old plugin library?